### PR TITLE
Allow for color: false to wipe the colorMap.

### PR DESCRIPTION
**Pull request by jdmarshall**
For issue #449

## 👀What is this pr about?

Summary of what this is about

### Added

- [x] feature1
- [x] feature2

## 🖼 Screenshots

| Before | After |
| ------ | ----- |
| 🧟     | 😎    |

## 👶 The naming of the PR

chore(deps): some dependency update
feat: Line Width is not limited, previously it was unlimited
fix(technical): Deployment issue with npm is fixed
fix: title names can be decorated

### CODE REVIEW

**Comment by ayonious** [comment]
This looks amazing

File: src/models/external-table.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export interface ComplexOptions {
   disabledColumns?: string[];
   computedColumns?: ComputedColumn[];
   rowSeparator?: boolean;
+  color? : boolean;
```
Comment (ayonious) [inline src/models/external-table.ts:35]: @jdmarshall Could you change this name to 
shouldDisableColors?: boolean; 
instead?

Comment (jdmarshall) [reply src/models/external-table.ts:35]: That doesn't seem to match up with the other arguments, but I'll try.

**Review by ayonious — APPROVED** [review]

**Pull request merged**